marlowe-dashboard-client/src/Capability/Marlowe.purs

```diff
@@ -213,15 +205,14 @@ instance manageMarloweAppM :: ManageMarlowe AppM where
             wallet = toFront $ view _cicWallet clientState
           walletContracts <- withExceptT Just $ ExceptT $ Contract.getWalletContractInstances wallet
           walletInfo <- withExceptT Just $ ExceptT $ Wallet.getWalletInfo wallet
```

Comment:
Understanding check: Do we drop this because we want to introduce more general solution for checks (per specific view sync or maybe based on some "app routing" logic etc.)?

Reply:
Yeah, as I mentioned in the general description, as part of SCP-3211 I want to have a more general way of getting this information. Probably setting it to the global store and having a heuristic for doing a global poll (either a fixed interval of X seconds or every time there is Y type of event, or similar). Also, we could later think if it makes sense for us to use the chainlink to listen for changes to a particular address.